v2/sourcedb-to-spanner/src/main/scripts/create_simple_shard_config.bash

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+# This is a sample script to help you generate the shard configuration json for a sharded bulk migration.
+# The script assumes that all shards share the same access credentials for the sake of simplicity.
+# Please modify the script as required for your use case.
+# Dependency: `bash` and `jq`
+
+# Steps to run this script
+# Step-1 update the variables to match your use case.
+# Step-2 Run the script.
+
+# Variables
+
+# Replace the below with the pat in GCS where you would like to upload the file.
+OUTPUT_PATH="gs://your-bucket/path/to/file.json"
+
+# Replace and add the Ips of all physical shards you would like to migrate in the array below.
+host_ips=(
+"ip.of.first.shard"
+"ip.of.second.shard"
+)
+# Port of your MySQL instance, generally 3306.
+port="3306"
+# Replace and add the names of all the logical databases you would like to migrate.
+# Note that all logical databases must have the same schema.
+database_names=("logicalDb01" "logicalDb02")
+# Replace with the actual DB credentials.
+username="<user-name>"
+password="<password>"
+
+# The script below will help you create the shard config json.
+
+# Main JSON Structure
+json='{
+  "configType": "dataflow",
+  "shardConfigurationBulk": {
+    "schemaSource": {
+      "dataShardId": "",
+      "host": "",
+      "user": "",
+      "password": "",
+      "port": "",
+      "dbName": ""
+    },
+    "dataShards": []
+  }
+}'
+
+# Populate dataShards array
+data_shards=()
+for host_ip in "${host_ips[@]}"; do
+    data_shard_id=$(echo "$host_ip" | tr '.' '-')
+    databases=()
+    for database_name in "${database_names[@]}"; do
```

Comment:
I think this assumes the same database names in all shards. That may not be the case

Reply:
That's right. I will add comments around the same.
It's done to keep it simple as in the most generic form, it becomes the json itself.

Reply:
Fixed in the current push.